src/transforms/rotate.jl

```diff
@@ -31,13 +34,13 @@ vector `u` to the plane passing through the origin with normal vector `v`.
 Rotate(Vec(1, 0, 0), Vec(1, 1, 1))
 ```
 """
-Rotate(u::Vec, v::Vec) = Rotate(uvrotation(u, v))
+Rotate(u::Vec, v::Vec) = Rotate(rotation_between(v, u))
```

Comment:
```suggestion
Rotate(u::Vec, v::Vec) = Rotate(rotation_between(u, v))
```
Here we want the rotation to rotate vector u to vector v, right?